app/views/home/index.html.slim

```diff
@@ -1,2 +1,37 @@
-h1 Home#index
-p Find me in app/views/home/index.html.slim
+.container
+  h1 Investment tracking
+
+.entries
+  .container-fluid
+    .row
+      .col-md-2
+      .col-md-6
+        p Your open investment time in hours:
+      .col-md-2
+        p Your rate to total worked hours:
+      .col-md-2
+        p Your used investment time:
+    .row
+      hr
+    - @sorted_entries.each do |entries|
+      .row
+        .col-md-2
+          p = entries['name'].force_encoding('ISO-8859-1').encode('UTF-8')
```

Comment:
I'd like if the encoding is already handled during the import of the data from Redmine. Then we can expect UTF-8 everywhere in our application.